addons/controllers/suite_test.go

```diff
@@ -142,13 +150,20 @@ var _ = BeforeSuite(func(done Done) {
 	Expect(err).ToNot(HaveOccurred())
 	Expect(dynamicClient).ToNot(BeNil())
 
+	tmpDir, err := os.MkdirTemp("/tmp", "webhooktest")
+	Expect(err).ToNot(HaveOccurred())
+	certPath = path.Join(tmpDir, "tls.cert")
+	keyPath = path.Join(tmpDir, "tls.key")
```

Comment:
Question: I assume the certificates specified here will not be valid in the first place?

Reply:
this are not certificates. this are just the fulll file path where the certificates will be written to.